core/web-assets/src/main/assets/js/apps/onms-requisitions/lib/views/service.html

```diff
@@ -0,0 +1,50 @@
+<div>
+<!-- Alejandro Galue (agalue@opennms.org) -->
+
+<div class="modal-header">
+  <h3 class="modal-title">Service</h3>
+</div>
+
+<div class="modal-body">
+  <form name="serviceForm">
+    <div class="form-group">
+      <label class="col-form-label" for="name">Name</label>
+      <input class="form-control" type="text" id="name" name="name" placeholder="A Service Name" ng-model="service.name" required valid-service
+             ng-class="{ 'is-invalid' : serviceForm.name.$invalid }">
+      <p ng-show="serviceForm.name.$invalid" class="invalid-feedback">A valid Service name is required, and must be unique within the interface.</p>
+    </div>
+
+    <hr>
+    <div class="clearfix">
+      <button type="button" class="btn btn-sm btn-secondary pull-right" id="add-metadata" ng-click="addMetaData()" ng-disabled="serviceForm.$invalid">Add Meta-Data</button>
```

Comment:
This is failing for me when creating a new requisition:
```
TypeError: Cannot read property 'push' of undefined
    at e.$scope.addMetaData (Service.js:68)
    at fn (eval at compile (angular.js:15126), <anonymous>:4:224)
```

Reply:
Fixed.